Makefile

```diff
@@ -31,18 +26,12 @@ dev: build
 test: build
 	@test -n "$$SPLUNK_ADDR" || { echo 'warning: SPLUNK_ADDR not set, creating new Splunk instances.  This will be slow.'; }
 	mkdir -p $(dir $(TESTREPORT))
-	gotestsum --junitfile $(TESTREPORT) -- -cover -v ./...
+	go clean -testcache || true
```

Comment:
```env GOCACHE=off gotestsum...```

Reply:
$ make test
mkdir -p ./
env GOCACHE=off gotestsum --junitfile test-results.xml --format standard-verbose -- -cover -v ./...
build cache is disabled by GOCACHE=off, but required as of Go 1.12

=== Errors
build cache is disabled by GOCACHE=off, but required as of Go 1.12

DONE 0 tests, 1 error in 0.015s
make: *** [test] Error 1